src/content/roadmap/account-abstraction/index.md

```diff
@@ -0,0 +1,95 @@
+
+The way wallets work would also change under EIP-4337. Instead of each wallet re-implementing common but complex safety logic, those functions would be outsourced to a global wallet contract known as the "entry point". This would handle operations such as paying fees and executing EVM code so that wallet developers can focus on providing excellent user experiences.
+
+[Read more on EIP-4337](https://medium.com/infinitism/erc-4337-account-abstraction-without-ethereum-protocol-changes-d75c9d94dc4a)
+[Read the EIP-4337 documentation](https://eips.ethereum.org/EIPS/eip-4337)
+
+</ExpandableCard>
+
+<ExpandableCard title="EIP-2938: changing the Ethereum protocol to support account abstraction">
+
+EIP-2938 aims to update the Ethereum protocol by introducing a new transaction type, `AA_TX_TYPE` that includes three fields: `nonce`, `target` and `data`, where `nonce` is a transaction counter, `target` is the entry point contract address and `data` is EVM bytecode. To execute these transactions, two new instructions (known as opcodes) have to be added to the EVM: `NONCE` and `PAYGAS`. The `NONCE` opcode tracks the transaction sequence and `PAYGAS` calculates and withdraws the gas required to execute the transaction from the contract's balance. These new features allow Ethereum to support smart contract wallets natively as the necessary infrastructure is built in to Ethereum's protocol.
+
+Note that EIP-2938 is currently not active. The community is currently favoring EIP-4337 because it does not require changes to the protocol.
+
+[Read more about EIP2938](https://hackmd.io/@SamWilsn/ryhxoGp4D#What-is-EIP-2938)
+[Read the EIP-2938 documentation](https://eips.ethereum.org/EIPS/eip-2938)
+
+</ExpandableCard>
+
+## When will account abstraction ship? {when-will-aa-ship}
+
+Smart contract wallets are already available, but more upgrades are required to make them as decentralized and permissionless as possible. EIP-4337 is a mature proposal that does not require any changes to Ethereum's protocol, so it is possible that this could be implemented quickly. Upgrades that alter Ethereum's protocol are currently not in active development, so may take much longer to ship. It is also very possible that account abstraction is achieved well enough by EIP-4337 that no protocol changes are ever required.
+
+## Further Reading {#further-reading}
+
+- [Watch the account abstraction panel discussion from Devcon Bogota](https://www.youtube.com/watch?app=desktop&v=WsZBymiyT-8)
+- [Watch "Why account abstraction is a game changer for dapps" from Devcon Bogota](https://www.youtube.com/watch?v=OwppworJGzs)
+- [Watch "Account abstraction ELI5" from Devcon Bogota](https://www.youtube.com/watch?v=QuYZWJj65AY)
+- [Read Vitalik's "Road to Account Abstraction" notes](https://notes.ethereum.org/@vbuterin/account_abstraction_roadmap#Transaction-inclusion-lists)
+- [Read Vitalik's blog post on social recovery wallets](https://vitalik.ca/general/2021/01/11/recovery.html)
```

Comment:
This is introducing a new convention for how we list further reading items by using action verbs here. Was this a conscious choice? 

Not opposed to this approach, but we've agreed that we'd like consistency across pages, so it feels like we should have a broader discussion about this, given there are site wide implications. 

cc: @konopkja

Reply:
updated to remove action verbs